app/navigation/Navigator.scala

```diff
@@ -52,6 +52,23 @@ class RootNavigator @Inject()() extends Navigator {
   override def defaultNormalMode: Call = controllers.routes.IndexController.onPageLoad()
 
   override def defaultCheckMode: Call = controllers.routes.IndexController.onPageLoad()
+
+  private def isBasicDetailsIncomplete(srn: Srn, userAnswers: UserAnswers): Boolean = {
+    val basicDetails: Option[Unit] = for {
+      _ <- userAnswers.get(CheckReturnDatesPage(srn)).flatMap {
+        case false => userAnswers.get(AccountingPeriods(srn))
+        case _ => Some(())
+      }
+      _ <- userAnswers.get(ActiveBankAccountPage(srn)).flatMap {
+        case false => userAnswers.get(WhyNoBankAccountPage(srn))
+        case _ => Some(())
+      }
+      _ <- userAnswers.get(HowManyMembersPage.bySrn(srn))
+      _ <- userAnswers.get(WhichTaxYearPage(srn))
```

Comment:
`WhichTaxYearPage` only gets populated on the overview page, should it be left out from this list here? https://github.com/hmrc/pension-scheme-return-frontend/blob/5f2c7a52f47c4b19c3720b532d2452598405d399/app/controllers/OverviewController.scala#L233